dspace-server-webapp/src/test/java/org/dspace/app/rest/CommunityCollectionPermissionIT.java

```diff
@@ -0,0 +1,919 @@
+        getClient(token).perform(get("/api/eperson/groups/" + adminGroup.getID() + "/subgroups"))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$._embedded.subgroups", Matchers.not(Matchers.hasItem(
+                            GroupMatcher.matchGroupWithName(group.getName())
+                        ))));
+
+        context.turnOffAuthorisationSystem();
+        configurationService.setProperty("core.authorization.community-admin.admin-group", true);
+        context.restoreAuthSystemState();
+
+    }
+
+    @Test
+    public void communityAdminRemoveChildGroupFromCommunityAdminGroupPropertySetToFalse() throws Exception {
+
+        context.turnOffAuthorisationSystem();
+        Group adminGroup = communityService.createAdministrators(context, parentCommunity);
+        authorizeService.addPolicy(context, parentCommunity, Constants.ADMIN, eperson);
+        Group group = GroupBuilder.createGroup(context).withName("testGroup").build();
+        context.restoreAuthSystemState();
+
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(
+            post("/api/eperson/groups/" + adminGroup.getID() + "/subgroups")
+                .contentType(MediaType.parseMediaType
+                    (org.springframework.data.rest.webmvc.RestMediaTypes
+                         .TEXT_URI_LIST_VALUE))
+                .content("https://localhost:8080/server/api/eperson/groups/" + group.getID()));
+
+        getClient(token).perform(get("/api/eperson/groups/" + adminGroup.getID() + "/subgroups"))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$._embedded.subgroups", Matchers.hasItem(
+                            GroupMatcher.matchGroupWithName(group.getName())
+                        )));
+
+        context.turnOffAuthorisationSystem();
+        configurationService.setProperty("core.authorization.community-admin.admin-group", false);
+        context.restoreAuthSystemState();
+
+        getClient(token).perform(delete("/api/eperson/groups/" + adminGroup.getID() + "/subgroups/" + group.getID()))
+                        .andExpect(status().isForbidden());
+
+        token = getAuthToken(admin.getEmail(), password);
+        getClient(token).perform(get("/api/eperson/groups/" + adminGroup.getID() + "/subgroups"))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$._embedded.subgroups", Matchers.hasItem(
+                            GroupMatcher.matchGroupWithName(group.getName())
+                        )));
+
+        context.turnOffAuthorisationSystem();
+        configurationService.setProperty("core.authorization.community-admin.admin-group", true);
+        context.restoreAuthSystemState();
+    }
+
+
+
+
+
+
+
+    @Test
+    public void communityAdminAddMembersToCollectionAdminGroupSuccess() throws Exception {
```

Comment:
I will rename my test to communityAdminCanManageCollectionSubmittersGroupAndAdminsGroupsTest and add 
this test inside my test 
https://github.com/DSpace/DSpace/blob/34b61f3b98eb22436a652f6270ff32e85cd6eb2f/dspace-server-webapp/src/test/java/org/dspace/app/rest/GroupRestRepositoryIT.java#L2323